.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "📖 Documentation"
+about: Suggest enhancement for devfile documentation
```

Comment:
maybe this is more complete? `Suggest an enhancement or report a bug for devfile documentation`

Reply:
how about `Suggest an update for devfile documentation`?

Reply:
updated